components/offscreen/offscreen.js

```diff
@@ -25,6 +26,7 @@ class Offscreen extends RtlMixin(LitElement) {
 		return css`
 			:host {
 				height: 1px;
+				inset-inline-start: -10000px;
```

Comment:
Technically, adding it here is not needed to solve the issue, because `d2l-offscreen` imports the `RTLMixin`.  Same with in the `sass` file, which doesn't need to worry about `shadowRoot`s. But I figured I'd put it in all three places now, for consistency - to highlight any issues early and to remind us when we go to remove the `left` and `right` solution.